src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -186,12 +210,66 @@ describe("Shipping", () => {
       const component = getWrapper(testProps)
       const mockCommitMutation = commitMutation as jest.Mock<any>
       mockCommitMutation.mockImplementationOnce((_environment, config) => {
+        component.update()
         expect(config.variables.input.shipping.name).toBe("Dr Collector")
```

Comment:
Let's just move the assertion out of this mock so you don't have to call `component.update()` and the test wouldn't look like a brain teaser.